Project/GNU/PKGBUILD

```diff
@@ -2,15 +2,15 @@
 # Maintainer: MediaArea.net SARL <info@mediaarea.net>
 # Contributor: hydro <hydro@freenet.de>
 
-pkgname=libmediainfo
+pkgname=('libmediainfo' 'python2-mediainfodll' 'python-mediainfodll')
```

Comment:
I think I prefer `python-mediainfo`, using `dll` here may be misleading (Windows thing) and this name comes from Debian IIRC, maybe not good to reproduce the same thing in other distros.

Reply:
Done